### PR TITLE
📚 Add availability information to docs

### PR DESCRIPTION
**Pull request by timroes**
Add information of which flavors of our product a specific doc applies to via frontmatter in the docs.

This will render out as follows:

The help icon links to our [feature comparison](https://airbyte.com/product/features) page.

The following values are allowed:

- `all` - mark the page applicable to all (will render out Cloud, OSS, Enterprise all checked)
- `oss-community` / `oss-enterprise` - mark the appropriate Self-managed flavor as checked (`oss-*` can be used as a shorthand for all Self-Managed flavors).
- `cloud` - mark Cloud as checked
- `cloud-teams` - Mark Cloud as checked and change the label to include "with Teams add-on"

TODO:
- Mark most docs for the correct flavor. so far just some initial pages have been marked for reviewing the overall logic.

### CODE REVIEW

File: docs/access-management/sso-providers/okta.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 sidebar_label: Okta
+flavors: oss-enterprise, cloud-teams
```
Comment (tealjulia) [inline docs/access-management/sso-providers/okta.md:3]: Would `supported_products` might be more self-explanatory to all?
No strong opinion.

Comment (timroes) [reply docs/access-management/sso-providers/okta.md:3]: But also so much to write :sob:

Comment (tealjulia) [reply docs/access-management/sso-providers/okta.md:3]: products?

Comment (timroes) [reply docs/access-management/sso-providers/okta.md:3]: Sounds good. Will change it to `products` :)

**Review by nataliekwong — APPROVED** [review]

**Pull request merged**